src/init/variables/VarResolver.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable tsdoc/syntax */
+import yargs from 'yargs';
+import { createErrorMessage } from '../..';
+import { AsyncHandler } from '../../util/handlers/AsyncHandler';
+import { modulePathPlaceholder } from '../../util/PathUtil';
+import type { VarComputer } from './VarComputer';
+
+const defaultConfig = `${modulePathPlaceholder}config/default.json`;
+const defaultVarConfig = `${modulePathPlaceholder}config/app-setup/vars.json`;
+
+export type YargsArgOptions = Record<string, yargs.Options>;
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export function setupYargvWithBaseArgs(yargv: yargs.Argv) {
+  // Values from componentsjs LogLevel type
+  const logLevels = [ 'error', 'warn', 'info', 'verbose', 'debug', 'silly' ];
```

Comment:
Can we import that from the logging module?

Reply:
Componentsjs defines it's own enum type currently. And thus it's cli expects values from that enum, which diverge from logging module. And as that is ts type,  but we want all those variants as strings in this case for cli-choices, we seems have to define them.

Reply:
We'd need to define this at `src/logging/LogLevel.ts` then.